### PR TITLE
[NeuralChat] refine dockerfile

### PR DESCRIPTION
**Pull request by XuehaoSun**
## Type of Change

refine Dockerfile and README.md

## Description

Remove conda in dockerfile
Dockerfile compatible with gpu nightly testing

## Expected Behavior & Potential Risk

no

## How has this PR been tested?

CI

## Dependency Change?

no

### CODE REVIEW

File: docker/Dockerfile_chatbot [hunk 1]
```diff
@@ -21,9 +21,9 @@
 #
 # ============================================================================
 # How to build: 
-#   docker build ./ -f Dockerfile_chatbot -t chatbot:latest
+#   docker build ./ -f Dockerfile -t chatbot_finetune:latest
```
Comment (chensuyue) [inline docker/Dockerfile_chatbot:24]: Dockerfile --> Dockerfile_chatbot

File: docker/Dockerfile_chatbot [hunk 2]
```diff
@@ -21,9 +21,9 @@
...
 # If you need to use proxy, please use the following command
-#   docker build ./ --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${http_proxy} -f Dockerfile_chatbot -t chatbot:latest
+#   docker build ./ --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${http_proxy} -f Dockerfile -t chatbot_finetune:latest
```
Comment (chensuyue) [inline docker/Dockerfile_chatbot:26]: Dockerfile --> Dockerfile_chatbot

File: docker/Dockerfile_chatbot (hunk 1 above)
Comment (XuehaoSun) [reply docker/Dockerfile_chatbot]: fixed. thanks.

File: docker/Dockerfile_chatbot (hunk 2 above)
Comment (XuehaoSun) [reply docker/Dockerfile_chatbot]: fixed. thanks.

**Comment by VincyZhang** [comment]
extension test: https://inteltf-jenk.sh.intel.com/job/ITREX-Chatbot-Finetune/87/

**Review by chensuyue — APPROVED** [review]

File: intel_extension_for_transformers/neural_chat/docker/inference/README.md [hunk 3]
```diff
@@ -5,38 +5,60 @@ Intel Neural Chat Inference Dockerfile installer for Ubuntu22.04
...
 Option 2: Download from docker hub.
-```
+```bash
 docker pull intel/ai-tools:itrex-chatbot 
 ```
 
 If you have downloaded models and dataset locally, just mount the files to the docker container using '-v'. Make sure using the absolute path for host_dir.
-```
+```bash
 docker run -it -v ${host_dir}:${mount_dir} ${IMAGE_NAME}:${IMAGE_TAG}
```
Comment (chensuyue) [inline intel_extension_for_transformers/neural_chat/docker/inference/README.md:25]: -v ${host_dir}:${mount_dir}, no need to mount, just use the copied repo.

**Review by chensuyue — APPROVED** [review]

**Pull request merged**